common/test/acceptance/pages/lms/dashboard.py

```diff
@@ -177,3 +177,16 @@ def pre_requisite_message_displayed(self):
         Verify if pre-requisite course messages are being displayed.
         """
         return self.q(css='section.prerequisites > .tip').visible
+
+    def click_username_dropdown(self):
+        """Click username dropdown"""
+        self.q(css='.dropdown').first.click()
+
+    @property
+    def username_dropdown_links(self):
+        """Return list username dropdown links"""
```

Comment:
Sorry for the confusion, but it turns out that I'm wrong about this. Single line comments should be as @muzaffaryousaf says:

https://github.com/edx/edx-platform/wiki/Python-Guidelines